CosmosContracts/juno-tools
# chore(assets): move brand svgs to public directory

Fixes #127

## Description

This PR moves the brand SVGs to the public directory to allow downloading assets.

## Changes

List any technical changes.

- [x] [move brand svgs to public directory](https://github.com/CosmosContracts/juno-tools/commit/e39b16cce8ce14bfe1b1bb59363a49616be73244)
- [x] [handle brand preview download behavior](https://github.com/CosmosContracts/juno-tools/commit/6ca9b12c65a40592c05d53787fafd4bfef9c911f)

## Screenshots

<img width="749" alt="image" src="https://user-images.githubusercontent.com/8220954/162274524-13e1c0c9-7f20-4d31-bd0f-b65be889d6e3.png">

## Testing Steps

As a reviewer, what steps should I take to verify this is working correctly?

- go to brand page and attempt downloading any asset

## Links

\-
